db/schema.rb

```diff
@@ -11,11 +11,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160301164601) do
+ActiveRecord::Schema.define(version: 20160310143553) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "badges", force: :cascade do |t|
+    t.string   "badge"
```

Comment:
What you're missing is the connection to the user account.  This `badges` table should have a `user_id` field.  From there should be `has_many` and `belongs_to` relationships in the `user.rb`, `badge.rb` models accordingly.